### DISCUSSION TITLE
Github webpages not loading on Firefox 117.0.1

### DISCUSSION THREAD

**Discussion by jkauk-n** [community/community#67058] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

This morning the Github webpages (any of them) stopped loading properly on Firefox 117.0.1 on my Ubuntu. Yesterday the pages worked. Parts of the page appear, but then I get loading icons that never clear. Github works normally on Chrome.

See the attached screenshots. My console is full of errors about:

```
Content-Security-Policy: The page’s settings blocked the loading of a resource at https://github.githubassets.com/assets/wp-runtime-33b747f52167.js (“script-src”).
```

and other resources.

I've tried loading the page with all extensions disabled, including ublock origin.

![github-bug2](https://github.com/community/community/assets/93518634/80b6f822-720d-43c6-a431-44e6195496c1)

![github-bug1](https://github.com/community/community/assets/93518634/e9c4d283-f5c8-44f6-a2fe-0d15d717e355)

**Comment by fumin** [comment]
Same here! Github team, roll up your sleeves and fix this ASAP!!!!!!!!

**Comment by sunaku** [comment]
This problem affects [one of my repos](https://github.com/sunaku/engrammer/tree/da4e827a88f725fdc288eaecaeebd9be20d34576#readme) (also compare with [the fixed version](https://github.com/sunaku/engrammer/tree/62ac091b3ba701a158536e202a6f907cc03bbf28#readme) of the same repo) but [not another one](https://github.com/sunaku/tamzen-font). 🤷  So it's not consistent, otherwise everyone would have noticed this by now. 🤔  Specifically, it appears that the `camo.githubusercontent.com` service isn't intercepting remote images anymore and replacing them with Github's cached versions.  Ditto for [local images bundled in the repo itself](https://github.com/orgs/community/discussions/68547).
